app/style/main.scss

```diff
@@ -103,10 +93,10 @@ $duration: .2s;
       left: -1vw;
       box-sizing: border-box;
       border-radius: 50%;
-      animation: pulse-ring 1.25s cubic-bezier(0.215, 0.61, 0.355, 1) infinite;
+      animation: pulse__ring 1.25s cubic-bezier(0.215, 0.61, 0.355, 1) infinite;
     }
 
-    @keyframes pulse-ring {
+    @keyframes pulse__ring {
```

Comment:
I don't think we need to use BEM naming for keyframe names. Well, we would just use a block naming convention since there would be no elements and I don't think there would be modifiers either but there may be.

But one thing we need to do is choose a name that will be unlikely to collide with any names the user might be using for their keyframes. We should stick with prefixing it with `xb-`.